.github/workflows/push-chart-ci.yaml

```diff
@@ -120,7 +120,11 @@ jobs:
             "hubble.relay.image.repository": "quay.io/${{ env.QUAY_ORGANIZATION_DEV }}/hubble-relay-ci",
             "hubble.relay.image.tag": "${{ steps.get-ref.outputs.sha }}",
             "clustermesh.apiserver.image.repository": "quay.io/${{ env.QUAY_ORGANIZATION_DEV }}/clustermesh-apiserver-ci",
-            "clustermesh.apiserver.image.tag": "${{ steps.get-ref.outputs.sha }}"
+            "clustermesh.apiserver.image.tag": "${{ steps.get-ref.outputs.sha }}",
+            "clustermesh.apiserver.kvstoremesh.image.repository": "quay.io/${{ env.QUAY_ORGANIZATION_DEV }}/kvstoremesh-ci",
+            "clustermesh.apiserver.kvstoremesh.image.tag": "${{ steps.get-ref.outputs.sha }}",
+            "clustermesh.apiserver.kvstoremesh.image.digest": "",
+            "clustermesh.apiserver.kvstoremesh.image.useDigest": false
```

Comment:
Why do these changes need to be put here, but the other images don't specify digest/useDigest?

Reply:
> Additionally, explicitly disable the digest, which is otherwise configured in stable branches.

It's configured that way so that if you pull the branch and install from Helm on that branch, it'll install the latest version from that branch. If we want to change this behaviour, we should change it globally, not for one specific image.

Reply:
Yeah, the other images are modified [here](https://github.com/cilium/cilium/pull/27646) (it's a different PR because it needs to be backported to different branches). The issue is that this workflow  overrides the tag to build a chart using the images from the current commit, but given that it does not remove the digest it will not work on stable branches (as the digest would be that of the latest release for that branch).

Reply:
With the ocurrent action, I had to manually disable the digest when using the development chart from a PR targeting v1.14 in https://github.com/cilium/cilium/pull/27232/commits/978071bdd353aa6add05c3c6318eca1b00e9bc03